HPXMLtoOpenStudio/tests/util.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-def get_ems_values(ems_objects, name)
+def get_ems_values(ems_objects, name, parse_more_operators = false)
```

Comment:
Do we really need this argument? I.e., would things break if we exclude this argument and always parse more operators? (And if so, why?)

Reply:
Yes it will break, basically the airflow tests. My first thought when it happened was that maybe it's because I am setting variables to be 1 if the statement.to_f is 0.0, but sometimes it's a true 0.0.